source/05_extending_alignak/alignak_backend.rst

```diff
@@ -0,0 +1,121 @@
+.. _extending/alignak_backend:
+
+===============
+Alignak backend
+===============
+
+The Alignak backend is a REST backend dedicated to the Alignak framework. It is used to:
+
+    - manage monitoring configuration (hosts, services, contacts, timeperiods...)
+
+        * creates and updates monitoring objects configuration
+        * user rights management
+        * template management for hosts and services
+
+    - manage retention
+
+        * loads and saves retention information for checks/hosts/services
+
+    - manage live states
+
+        * creates and updates states for hosts and services
+        * manages acknowledges, downtimes, re-checks
+
+    - feed time series databases with performance data
+
+        * get performance data from the elements check to feed Graphite and/or Influx
+
+    - system activity log
+
+        * store all the system acitivity in a timestamped log
+
+
+More information is available in the `project repository <https://github.com/Alignak-monitoring-contrib/alignak-backend>`_.
+
+The backend full documentation is available on http://alignak-backend.readthedocs.io/en/latest/intro.html
+
+REST API
+========
+
+The Alignak backend, based on the Python Eve frameworks, implements a complete REST API to manage the monitored objects data model.
+
+The detailed API is available in the backend documentation.
+
+
+Using the backend with Alignak
+==============================
+
+Installing the Alignak backend is an easy operation.
+The different installation methods are detailed in the documentation but the most easy way is to::
+
+    pip install alignak-backend
+
+
+Once installed, the backend is ready to run and you can start with::
+
+    ./bin/run.sh
+
+
+Configuring the Alignak backend is made easy thanks to a *settings.json* file.
+The default configuration makes the backend run on the local interface, on port 5000, and it creates an admin user that can be used to log-in.
+
+**Note**: to be updated after the backend packaging ...
+
+Once installed, the backend needs to be related to Alignak daemons. This is achieved thanks to the `Alignak backend`_ modules.
+
+A backend installation tutorial exists on the `SysAdmin.cool web site <http://sysadmin.cool/>`_.
+
+Feeding the backend
+===================
+
+On start, the backend creates some elements if they do not yet exist:
+
+    * a default super administrator user named `admin`
```

Comment:
+ password 'admin'
It's better to add the default password because users will always ask us